.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,38 @@
+name: Build and Push Docker Image
+
+on:
+  pull_request:
+    branches:
+      - custom-main
+  push:
+    branches:
+      - custom-main
```

Comment:
is there a reason why we have all the branches from the forked repo?

Reply:
No reason.